src/listeners/ticket/create-modal-submit-validator.ts

```diff
@@ -0,0 +1,183 @@
+            !interaction.inGuild()
+        ) {
+            return;
+        }
+
+        await interaction.deferReply({ ephemeral: true });
+
+        const title = interaction.fields.getTextInputValue(InputId.Title).trim();
+        if (!title) {
+            return this.sendValidationFailure(
+                interaction,
+                'Title must contain at least 1 non-whitespace character'
+            );
+        }
+
+        const exp = interaction.fields.getTextInputValue(InputId.Explanation).trim();
+        if (!exp) {
+            return this.sendValidationFailure(
+                interaction,
+                'Description must contain at least 1 non-whitespace character'
+            );
+        }
+
+        const guildSettings = await settingsManager.get(interaction.guildId);
+        const guild = await interaction.client.guilds.fetch(interaction.guildId);
+        const member = await guild.members.fetch(interaction.member.user.id);
+
+        const ticketChannelId = guildSettings?.ticketChannel;
+        if (!ticketChannelId) {
+            return;
+        }
+        const ticketChannel = await guild.channels.fetch(
+            ticketChannelId
+        );
+        if (ticketChannel?.type !== ChannelType.GuildText) {
+            return;
+        }
+
+        const latestTicket = await ticketLogs.findOne(
+            { '_id.guild': interaction.guildId! },
+            { sort: { 'number': -1 } }
+        );
+
+        
+        const latestTicketNumber = latestTicket ? latestTicket.number : 0;
+        const ticketID = String(latestTicketNumber + 1).padStart(6, '0');
+
+        const threadName = `${title} - #${ticketID}`;
+
+        let thread = await ticketChannel.threads.create({
+            name: threadName,
+            reason: `Ticket for ${interaction.user.id}`,
+            type: ChannelType.PrivateThread,
+            invitable: false,
+        });
+
+        const roles = await guild.roles.fetch();
+
+        await thread.send(
+            [
+                interaction.user,
+                ...roles
+                    .filter(role =>
+                        role
+                            .permissionsIn(ticketChannel)
+                            .has(PermissionFlagsBits.ManageThreads)
+                    )
+                    .values(),
+            ].join(' ')
+        );
+
+        const ticketCreate = await thread.send({
+            embeds: [
+                new EmbedBuilder()
+                    .setColor(Color.Yellow)
+                    .setAuthor({
+                        name: interaction.user.tag,
+                        url: userUrl(interaction.user.id),
+                        iconURL: (interaction.member instanceof GuildMember
+                            ? interaction.member
+                            : interaction.user
+                        ).displayAvatarURL(),
+                    })
+                    .setTitle(`${title}`)
```

Comment:
`title` is already a string, so we can use it directly.
```suggestion
                    .setTitle(title)
```